test-js/integration.js

```diff
@@ -192,6 +192,13 @@ describe("integration tests", function() {
     assert.exists(path.join("output", "Main", "index.js"));
   }));
 
+  notWindowsIt("pulp build with psc-package", run(function*(sh, pulp, assert, temp) {
```

Comment:
Ah right, I see. Okay in that case, I'd prefer to either install `psc-package` from source in pulp's appveyor CI script, or set up appveyor CI for psc-package as described in that issue. What do you think? The former is probably easier for now.